src/helpers/gas.ts

```diff
@@ -0,0 +1,35 @@
+const PRIORITY_FEE_INCREMENT = 0.5;
+const PRIORITY_FEE_THRESHOLD = 0.15;
+
+export const getTrendKey = (trend: number) => {
+  if (trend < -0.3) {
+    return 'falling';
+  } else if (trend >= -0.3 && trend <= 0.3) {
```

Comment:
could we get rid of line 7 and 8 and instead use an `else` at the end for `stable`